monai/transforms/utils_pytorch_numpy_unification.py

```diff
@@ -50,3 +52,23 @@ def in1d(x, y):
     if isinstance(x, np.ndarray):
         return np.in1d(x, y)
     return (x[..., None] == torch.tensor(y, device=x.device)).any(-1).view(-1)
+
+
+def clip(a: NdarrayOrTensor, a_min, a_max) -> NdarrayOrTensor:
+    """`np.clip` with equivalent implementation for torch."""
+    result: NdarrayOrTensor
+    if isinstance(a, np.ndarray):
+        result = np.clip(a, a_min, a_max)
+    else:
+        result = torch.clip(a, a_min, a_max)
+    return result
+
+
+def percentile(x: NdarrayOrTensor, q):
+    """`np.percentile` with equivalent implementation for torch."""
+    result: NdarrayOrTensor
+    if isinstance(x, np.ndarray):
+        result = np.percentile(x, q)
+    else:
+        result = torch.quantile(x, q / 100.0)
```

Comment:
torch.quantile is available since v1.7, perhaps we can use optional import here and skip the test for v1.6

Reply:
@wyli , what should we use for PyTorch 1.5 and 1.6?

Thanks.

Reply:
Added our own implementation for pre 1.7.